### PR TITLE
Update AA, image-rs td-shim

### PR DESCRIPTION
**Pull request by surajssd**
- image-rs & AA: 917984103e1029d8ff93d91df3c973761910508c
- td-shim: 4ecf76ab8bc57c2e857937487d1923961c2b6769

> **Note**: This PR is for reducing delta to reduce work during release and find problems early on.

### CODE REVIEW

**Comment by fidencio** [comment]
@surajssd, Kata Containers has a bunch of checks that, at first, may be annoying, but you'll get used to it.

One if the checks we have is that an issue must be created for a PR, and then it must be linked to the PR by adding `Fixes: #NNNN` to the commit message, with NNNN being the number of the issue you created.

This is the reason why the "Commit Message Check" is failing now.  Mind to create an issue and adjust the commit message?

**Comment by surajssd** [comment]
> One if the checks we have is that an issue must be created for a PR, and then it must be linked to the PR by adding `Fixes: #NNNN` to the commit message, with NNNN being the number of the issue you created.

Yep, done #7580.

**Comment by jepio** [comment]
For the dragonball failure this might be needed:
- https://github.com/kata-containers/kata-containers/pull/7581

**Comment by surajssd** [comment]
Oh nothing happened, it was worth trying 😆

**Comment by surajssd** [comment]
@fidencio @jepio can i get permissions to run tests on my own?

File: .github/workflows/static-checks.yaml [hunk 1]
```diff
@@ -79,6 +79,10 @@ jobs:
         echo "Set environment variables for the libseccomp crate to link the libseccomp library statically"
         echo "LIBSECCOMP_LINK_TYPE=static" >> $GITHUB_ENV
         echo "LIBSECCOMP_LIB_PATH=${libseccomp_install_dir}/lib" >> $GITHUB_ENV
+    - name: Install dependency
+      run: |
+        sudo apt update
+        sudo apt install -y libdevmapper-dev
```
Comment (jepio) [inline .github/workflows/static-checks.yaml:85]: This should go as a PR to github.com/kata-containers/tests and then you can include this for testing with this repo by adding an entry in your commit: https://github.com/kata-containers/tests#breaking-compatibility.

Comment (surajssd) [reply .github/workflows/static-checks.yaml:85]: Umm, I don't understand. All the static checks have passed, though?

If this should go to the tests repo, how does it benefit the static checks? The GHA for static checks shows that there is no dependency on anything outside for those checks.

**Comment by surajssd** [comment]
Can someone trigger the test run?

**Comment by surajssd** [comment]
Triggering it again: 
image

File: .github/workflows/static-checks.yaml (hunk 1 above)
Comment (surajssd) [reply .github/workflows/static-checks.yaml]: Done!

**Comment by fidencio** [comment]
Cloud HJypervisor passed but it seems it was not updated here: http://10.112.240.228:8080/job/kata-containers-CCv0-cloud-hypervisor-TDX-CRI_CONTAINERD_K8S-PR/1082/

Screenshot from 2023-08-17 16-44-39

**Comment by fidencio** [comment]
The TDX one failed for reasons that do not seem to be related to the test, just networking issues:
```
16:18:39 error: failed to download from `[https://crates.io/api/v1/crates/prost-build/0.8.0/download`](https://crates.io/api/v1/crates/prost-build/0.8.0/download%60)
16:19:18 
16:19:18 Caused by:
16:19:18   [28] Timeout was reached (download of `prost-build v0.8.0` failed to transfer more than 10 bytes in 30s)
16:19:18 make: *** [Makefile:397: target/x86_64-unknown-linux-musl/release/containerd-shim-kata-v2] Error 101
16:19:18 make[1]: *** [tools/packaging/kata-deploy/local-build/Makefile:67: cc-shim-v2-tarball-build]
```

I've retriggered the tests.

**Comment by fidencio** [comment]
Both QEMU and CLH passed, so I'm triggering a `/test` here, and giving my approbal.

**Review by fidencio — APPROVED** [review]
lgtm, thanks for working on this this early, @surajssd!

**Comment by jepio** [comment]
@surajssd the s390x job is failing (http://jenkins.katacontainers.io/job/kata-containers-CCv0-ubuntu-20.04-s390x-PR/777/console):
```
10:17:12 error: failed to run custom build command for `devicemapper-sys v0.1.5`
10:17:12 
10:17:12 Caused by:
10:17:12   process didn't exit successfully: `/home/jenkins/workspace/kata-containers-CCv0-ubuntu-20.04-s390x-PR/go/src/github.com/kata-containers/kata-containers/src/agent/target/release/build/devicemapper-sys-5544dfc02473b694/build-script-build` (exit status: 101)
10:17:12   --- stderr
10:17:12   thread 'main' panicked at 'Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"', /home/jenkins/.cargo/registry/src/github.com-eae4ba8cbf2ce1c7/bindgen-0.63.0/./lib.rs:2338:31
```

Reading the log it's failing here: https://github.com/kata-containers/tests/blob/main/.ci/jenkins_job_build.sh#L223-L242.

**Comment by surajssd** [comment]
@jepio yep on it. In the last few runs the s390x pipeline didn't even trigger.

**Comment by surajssd** [comment]
@jepio PTAL: https://github.com/kata-containers/tests/pull/5751

**Comment by stevenhorsman** [comment]
SEV test 3 failed, so I've re-queued

**Comment by fidencio** [comment]
Let me go ahead with this one.

**Pull request merged**